shared/src/main/scala/org/http4s/websocket/WebsocketBits.scala

```diff
@@ -138,5 +138,37 @@ object WebsocketBits {
     }
   }
 
+  sealed trait InvalidCloseDataException extends RuntimeException
+  class InvalidCloseCodeException(i: Int) extends InvalidCloseDataException
+  class ReasonTooLongException(s: String) extends InvalidCloseDataException
+
+  private def toUnsigned(x: Int) = Array[Byte](((x >> 8)& 0xFF).toByte, (x & 0xFF).toByte)
```

Comment:
ffti: I would call this `toUnsignedShort`

Should there be   a space between `8)& >>`?